microsoft/testsuites/mdatp/mdatp.py

```diff
@@ -22,6 +24,59 @@
 EXIT_ONBOARD_INFO_FOUND = 255
 
 
+def working_path_workaround(node: Node) -> None:
+    # EXAMPLE WORKAROUND:
+    # - Get the mount point for the path.
+    # - Check if the mount point is mounted noexec.
+    # - if yes, remount with exec flag
+    working_path = node.get_working_path()
+    mount_info = node.tools[Df].get_mount_info_for_dir(
+        directory=working_path.as_posix()
+    )
+    if not mount_info:
+        node.log.warn(
```

Comment:
Don't use log.warn. Use log.info or raise exception.

Reply:
Please use info level.
The warn method is implemented in Python package, not in LISA. In the practical, most people don't care warn level in the routine pipelines. When developing a new case, it needs to read every new log line to make sure it's clear to others.

Reply:
>  In the practical, most people don't care warn level in the routine pipelines.

I'll stop bringing this up, maybe I can make a feature request and ask people who give me feedback about LISA's logging to upvote it 😜

Reply:
I got feedback from a team, they use ado pipeline, and look at their error/warn lines. They don't want LISA has error/warning lines, which are not fixed.

Reply:
ok